### PR TITLE
Save page views

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: home/api.py [hunk 1]
```diff
@@ -28,6 +34,14 @@ class ContentPagesViewSet(PagesAPIViewSet):
...
         super(ContentPagesViewSet, self).get(self, request, *args, **kwargs)
 
+    def detail_view(self, request, pk):
+        ContentPage.objects.get(id=pk).save_page_view(request.query_params)
+        return self._detail_view(request, pk)
+
+    @method_decorator(cache_page(60 * 60 * 2))
+    def _detail_view(self, request, pk):
+        return super().detail_view(request, pk)
+
```
Comment (erikh360) [inline home/api.py:44]: I did it this way to log a page view even if returns a cached page.

File: home/api.py [hunk 2]
```diff
@@ -28,6 +28,10 @@ class ContentPagesViewSet(PagesAPIViewSet):
     def get(self, request, *args, **kwargs):
         super(ContentPagesViewSet, self).get(self, request, *args, **kwargs)
 
+    def detail_view(self, request, pk):
```
Comment (erikh360) [inline home/api.py:31]: The caching was messing with the tests so I removed it

File: home/models.py [hunk 3]
```diff
@@ -286,3 +300,14 @@ class ContentPageRating(models.Model):
     helpful = models.BooleanField()
     comment = models.TextField(blank=True, default="")
     data = models.JSONField(default=dict, blank=True, null=True)
+
+
+class PageView(models.Model):
+    timestamp = models.DateTimeField(auto_now=True)
```
Comment (codiebeulaine) [inline home/models.py:306]: isn't it auto_now_add?

**Review by codiebeulaine — APPROVED** [review]

File: home/models.py (hunk 3 above)
Comment (erikh360) [reply home/models.py:306]: yup

**Pull request merged**